lib/sub-process.ts

```diff
@@ -13,7 +13,7 @@ export function execute(
   if (options && options.cwd) {
     spawnOptions.cwd = options.cwd;
   }
-  args = quoteAll(args, spawnOptions);
+  args = quoteAll(args, { flagProtection: false });
```

Comment:
`shell: true` is [deprecated](https://nodejs.org/docs/latest-v24.x/api/deprecations.html#dep0190-passing-args-to-nodechild_process-execfilespawn-with-shell-option-true) behavior. The `shescape` doesn't take the argument anyway.